oteapi_dlite/strategies/function.py

```diff
@@ -113,6 +119,25 @@ def get(
         config = self.function_config.configuration
         cacheconfig = config.datacache_config
 
+        for addition in config.global_configuration_additions.storage_path:
```

Comment:
This should be fractured out to a utility function

Reply:
Agreed. Also, I originally used `setattr()`, which would make this much more elegant, but as I couldn't get it to work at all, I couldn't check whether it was OK or not.

I think in the great picture of things, I will simply remove this functionality and configuration option altogether - what do you think?

Reply:
I have removed it altogether now.